test/fixtures/transaction.json

```diff
@@ -164,8 +164,7 @@
           {
             "hash": "0000000000000000000000000000000000000000000000000000000000000000",
             "index": 4294967295,
-            "data": "032832051c4d696e656420627920416e74506f6f6c20626a343a45ef0454c5de8d5e5300004e2c0000",
-            "sequence": 4294967295
+            "data": "032832051c4d696e656420627920416e74506f6f6c20626a343a45ef0454c5de8d5e5300004e2c0000"
           }
```

Comment:
Just build all these unsigned tx's, and have the tests check it comes up with the same witness tx?